server/src/routes/v1/apps/handlers/createApp.js

```diff
@@ -154,13 +154,40 @@ module.exports = {
             }
 
             const organisationId = organisation.id
-            const requestUserId = currentUserId
+            let ownerUserId = currentUserId
             const developerUserId = appDeveloper.id
 
+            if (appJsonPayload.owner) {
+                const { email, name } = appJsonPayload.owner
+                const appOwner = await getUserByEmail(email, knex)
+
+                if (appOwner == null) {
```

Comment:
should use strict equality?

Reply:
I normally use strict yeah, but this catches undefined as well. I can change it though